sdk/clientcore/http-okhttp3/src/test/java/io/clientcore/http/okhttp3/OkHttpHttpClientDeadlockTests.java

```diff
@@ -41,7 +42,7 @@ public class OkHttpHttpClientDeadlockTests {
 
     @BeforeEach
     public void startTestServer() {
-        server = new LocalTestServer((req, resp, responseBody) -> {
+        server = new LocalTestServer(HttpProtocolVersion.HTTP_1_1, false, (req, resp, responseBody) -> {
```

Comment:
do we have any HTTP/2 tests?

Reply:
we do! in [JdkHttp2ClientTestIT.java](https://github.com/Azure/azure-sdk-for-java/pull/45758/files#diff-6ed2b18bb90c3e006bcb84adea7708129655339b76573d818d8157d3cd3c9be5)